CRM/Core/BAO/Address.php

```diff
@@ -774,7 +774,7 @@ public static function parseStreetAddress($streetAddress, $locale = NULL) {
     // the DB to fatal
     $fields = CRM_Core_BAO_Address::fields();
     foreach ($fields as $fieldname => $field) {
-      if (!empty($field['maxlength']) && strlen(CRM_Utils_Array::value($fieldname, $parseFields)) > $field['maxlength']) {
+      if (!empty($field['maxlength']) && strlen(CRM_Utils_Array::value($fieldname, $parseFields, '')) > $field['maxlength']) {
```

Comment:
We're trying to get rid of this function. Can it just be `$parseFields[$fieldname] ?? ''`. We know $parseFields is an array so don't have to think about that goofiness.

Reply:
Updated

Reply:
There were a couple others lower down - will attack them later.